com.binarynonsense.acbr.yml

```diff
@@ -0,0 +1,94 @@
+app-id: com.binarynonsense.acbr
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+command: run.sh
+separate-locales: false
+rename-icon: acbr-comic-book-reader
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=home
+  - --share=network
+  # needed to access the gamepad
+  - --device=all
+  - --filesystem=/run/udev:ro
+  # needed to avoid some crashes caused by sharp in some systems
+  - --env=G_SLICE=always-malloc
+  # use same mouse cursors as host, to avoid weird cursor sizing under Wayland with mixed DPI
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+build-options:
+  append-path: /usr/lib/sdk/node20/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: acbr
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/acbr/flatpak-node/cache
+        npm_config_cache: /run/build/acbr/flatpak-node/npm-cache
+        npm_config_offline: "true"
+    build-commands:
+      - jq '.build.linux.target="dir"' <<<$(<package.json) > package.json
+      - jq '.build.files=["!flatpak-node${/*}"]' <<<$(<package.json) > package.json
+      - npm install --offline
+      - chmod +x node_modules/7zip-bin/linux/x64/7za
+      - npm run build:linux
```

Comment:
now that t is building from source, remove the flathub.json to build for aarch64?